starters/blog/src/components/seo.js

```diff
@@ -28,47 +28,50 @@ function SEO({ description, lang, meta, title }) {
   const metaDescription = description || site.siteMetadata.description
 
   return (
-    <Helmet
-      htmlAttributes={{
-        lang,
-      }}
-      title={title}
-      titleTemplate={`%s | ${site.siteMetadata.title}`}
-      meta={[
-        {
-          name: `description`,
-          content: metaDescription,
-        },
-        {
-          property: `og:title`,
-          content: title,
-        },
-        {
-          property: `og:description`,
-          content: metaDescription,
-        },
-        {
-          property: `og:type`,
-          content: `website`,
-        },
-        {
-          name: `twitter:card`,
-          content: `summary`,
-        },
-        {
-          name: `twitter:creator`,
-          content: site.siteMetadata.author,
-        },
-        {
-          name: `twitter:title`,
-          content: title,
-        },
-        {
-          name: `twitter:description`,
-          content: metaDescription,
-        },
-      ].concat(meta)}
-    />
+    <HelmetProvider>
```

Comment:
As a notice, `gatsby-plugin-react-helmet-async` has a [gatsby-browser](https://github.com/Collabsalot/gatsby-plugin-react-helmet-async/blob/master/src/gatsby-browser.tsx) / [gatsby-ssr](https://github.com/Collabsalot/gatsby-plugin-react-helmet-async/blob/master/src/gatsby-ssr.tsx) script that adds this HelmetProvider with wrapRootElement so I believe this won't be needed in these component files, but it is just added as part of the gatsby plugin itself.

Reply:
Ok, good to know. Thanks! I just committed a correction.